fvm/fvm.go

```diff
@@ -50,21 +49,6 @@ func (vm *VirtualMachine) Run(ctx Context, proc Procedure, v state.View, program
 		state.WithMaxInteractionSizeAllowed(ctx.MaxStateInteractionSize))
 	sth := state.NewStateHolder(st)
 
-	defer func() {
```

Comment:
I guess this was removed because there is similar logic on the Cadence side now?

Reply:
This error no loner exists from the cadence side.